lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -207,43 +207,50 @@ redistribute_policy:
     config_set_append: ' <state> redistribute <protocol> route-policy <policy>'
 
 #config in vrf context
+# TODO: fix these for XR
 route_target_both_auto:
```

Comment:
fyi, the rd / route-target props are in the process of moving from bgp_af to vrf_af in the develop branch (PR #167) so you may want to just leave a breadcrumb on them for now. We plan to pull a release branch next tuesday and then you can start the fun of merging with develop.